tests/Export/Adapters/AttributeAdapterTest.php

```diff
@@ -0,0 +1,830 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Tests\Export;
+
+use FINDOLOGIC\Export\Data\Attribute;
+use FINDOLOGIC\Export\Data\Property;
+use FINDOLOGIC\Export\XML\XMLItem;
+use FINDOLOGIC\FinSearch\Exceptions\Export\Product\AccessEmptyPropertyException;
+use FINDOLOGIC\FinSearch\Exceptions\Export\Product\ProductHasNoCategoriesException;
```

Comment:
As this exception is an unused import, it suggests that you forgot about the test for it.
This exception was tested in `testProductAndVariantHaveNoCategories`